operations/deployment/terraform/modules/variables.tf

```diff
@@ -94,6 +94,11 @@ variable "create_vpc" {
   description = "Attach public IP to the EC2 instance"
 }
 
+variable "provided_vpc" {
```

Comment:
Assume you're using https://registry.terraform.io/modules/terraform-aws-modules/vpc/aws/latest#usage module (has `1.5M` downloads/month BTW) and try to come with a solution from the perspective that he VPC is managed by that module.

Reply:
`vpc_id` sounds more expected from the naming perspective.
https://github.com/terraform-aws-modules/terraform-aws-vpc/blob/aa61bc4346e1c430df8ec163ae9799d57df4af20/outputs.tf#L1
Then, for GH Actions inputs it would be `aws_vpc_id` variable for the user to set an existing VPC ID.


---

Outside of the VPC we'll need to provide the subnet id `aws_subnet_id` for the EC2 instance in case of VPC is managed outside of the GH Action.
For instance, https://registry.terraform.io/modules/terraform-aws-modules/vpc/aws/latest#usage creates `private_subnets`, `public_subnets` and so on, so user would just pass the ID of desired subnet to the GH Action.

Reply:
I agree the var should be `vpc_id`